CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 Changelog
 ---------
 
-master
-******
+0.1
```

Comment:
To be honest I think we do not need to include all these pull requests before the first release.

Reply:
Ah, ok. So, in general, we want this as "master" for the next release, right? Then it should be adjusted for the version right before the tag. Will change the release script to be agnostic of the header type. So I could undo this commit. What do you think?

Reply:
I think undoing it would be good. We should just right the version string in here by hand in the commit we tag as the next version (I think in general taking the version from the git tag rather than a string in CHANGELOG is safer).

Reply:
The version is taken from the git tag, but the header in CHANGELOG has to equal that so we can find the relevant CHANGELOG entries to use as the release description.

Reply:
@rgieseke @znicholls Nevertheless, does it make sense to collect the "changes" before any release?

Reply:
> The version is taken from the git tag, but the header in CHANGELOG has to equal that so we can find the relevant CHANGELOG entries to use as the release description.

Ah ok then. That sounds good, as long as it only happens on tags (so it's not complaining when the header is just `master`), is that possible?

Reply:
yes, will look for the header with the version, the rest is ignored.